Tests/DependencyInjection/Compiler/TwigFormPassTest.php

```diff
@@ -5,6 +5,9 @@
 use Norzechowicz\AceEditorBundle\DependencyInjection\Compiler\TwigFormPass;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 
+/**
+ * @author azzra <azzra@users.noreply.github.com>
```

Comment:
actually I would like to get rid of all @author annotations, it make no sense since we are using github and all authors are visible